jobs/integr8ly/poc-upgrade.yaml

```diff
@@ -0,0 +1,95 @@
+---
+
+- job:
+    name: poc-upgrade
+    project-type: pipeline
+    description: "Installs Integreatly remotely from Jenkins slave to QE Pony cluster."
+    sandbox: false
+    concurrent: true
+    parameters:
+        - string:
+            name: REPOSITORY
+            default: https://github.com/integr8ly/installation.git
+            description: "Repository of the Integreatly installer"
+        - string:
+            name: BRANCH
+            default: 'master'
+            description: "Branch of the installer repository"
+        - string:
+            name: CLUSTER_URL
+            description: "URL of cluster for installation."
+        - string:
+            name: ANSIBLE_USER
+            default: root
+            description: "User for Ansible to access the master node of target cluster"
+        - string:
+            name: MASTER_URLS
+            description: "Comma separated list of URLs for master nodes of target cluster to be used in Ansible inventory file"
+        - string:
+            name: OC_USER
+            description: "OpenShift user for QE cluster."
+        - string:
+            name: OC_PASSWORD
+            description: "OpenShift password for QE cluster."
+    dsl: |
+        timeout(60) { ansiColor('gnome-terminal') { timestamps {
```

Comment:
the same as in pds-upgrade. 60 minutes is probably too long timeout for an upgrade